docs/explanation/service-start-order.md

```diff
@@ -7,4 +7,8 @@ The `before` option is a list of services that this service must start before (i
 ```{include} /reuse/service-start-order.md
    :start-after: Start: Service start order note
    :end-before: End: Service start order note
-```
\ No newline at end of file
+```
+
+## Service auto-restart
+
+By default, if a service exits, Pebble will restart it after the backoff delay, which defaults to half a second. For more information, see [start command](/reference/cli-commands/start.md).
```

Comment:
Would it be possible to split the sentence to make it easier to read?

I don't think "By default" is critical, so this would work for me:

```
If a service exits, Pebble will restart it after the backoff delay, which defaults to half a second. For more information, see [start command](/reference/cli-commands/start.md).
```

But if it's important to mention this is the default behaviour, how about:

```
By default, if a service exits, Pebble will restart it after the backoff delay. The backoff delay defaults to half a second. For more information, see [start command](/reference/cli-commands/start.md).
```

Reply:
I'm not actually sure we need this change, or that it belongs here. This page is "Service start order", and this is more general than that. I think we should revert this change.

Though perhaps we can instead link "waits 1 second" above in the first paragraph to the new `pebble start` reference instead?

Reply:
I'd be good with removing the "Service auto-restart" heading + text, in the spirit of keeping the page focused on one topic. Good idea to link "waits 1 second" to the `start` command reference